lib/mix/test/mix/rebar_test.exs

```diff
@@ -33,20 +33,20 @@ defmodule Mix.RebarTest do
     end
   end
 
-  test "load rebar config" do
+  test "load Rebar config" do
     path = MixTest.Case.fixture_path("rebar_dep")
     config = Mix.Rebar.load_config(path)
     assert config[:sub_dirs] == ['apps/*']
     assert config[:SCRIPT] == 'rebar.config.script'
   end
 
-  test "execute rebar.config.script on dependency directory" do
+  test "execute Rebar.config.script on dependency directory" do
```

Comment:
I think that's a finename and should be kept intact.
I assume same valid for a test above.